generated/nidigital/nidigital/session.py

```diff
@@ -2346,7 +2867,15 @@ def read_static(self):
         nidigital.Session repeated capabilities container, and calling this method on the result.
 
         Returns:
-            data (list of enums.PinState):
+            data (list of enums.PinState): The returned array of pin states read from the channels in the **channelList**. Data is returned in the order you specify in the **channelList**. If a site is disabled, the method does not return data for that site. If you are using a list of pin names to read data from multiple instruments, use the SortPinResultsBySiteViUInt8 method to order and combine the data to match the **channelList**. You can also use the GetResultsPinMapInformation method to obtain a sorted list of returned sites and channels.
```

Comment:
Same comment about `SortPinResultsBySiteViUInt8` and `GetResultsPinMapInformation` as in `get_fail_count()`

Reply:
Done